### PR TITLE
ROC-6941 Updated content of ccj amount page

### PR DESCRIPTION
**Pull request by sudheer-ajax**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-6941

### Change description
Change content on judgment amount page to remove expectation of repayment

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/common/components/ccj/paid-amount-summary.njk [hunk 1]
```diff
@@ -35,7 +37,7 @@
             <th scope="row">{{ t('Amount you’re settling for') }}</th>
             <td class="numeric last">
               <span class="font-small">{{ amountSettledFor | numeral }}</span>
-            </td>
+            </td><x></x>
```
Comment (robertparkinson) [inline src/main/common/components/ccj/paid-amount-summary.njk]: what is this tag for?

File: src/main/common/components/ccj/paid-amount-summary.njk [hunk 2]
```diff
@@ -10,9 +10,11 @@
 
     <div class="form-group">
       {% set total = claim.totalAmountTillToday if amountSettledFor === undefined else amountSettledFor + (claimFee | pennies2pounds) %}
-
-      <p>{{ t('The judgment will order the defendant to pay {{ totalAmount }}, including your claim fee and any interest.', { totalAmount: total | numeral }) }}</p>
-      <p>{{ t('This means you´ll recover the {{ totalAmount }} as follows:', { totalAmount: total | numeral }) }}</p>
+      {% if interestDetails %}
+        <p>{{ t('The judgment will order the defendant to pay {{ totalAmount }}, including your claim fee and any interest, as shown in this table:', { totalAmount: total | numeral }) }}</p>
+      {% else %}
+        <p>{{ t('The judgment will order the defendant to pay {{ totalAmount }}, including your claim fee, as shown in this table:', { totalAmount: total | numeral }) }}</p>
```
Comment (KiranVarma02) [inline src/main/common/components/ccj/paid-amount-summary.njk]: are these 2 of statements same? or am i missing something?

Comment (sudheer-ajax) [reply src/main/common/components/ccj/paid-amount-summary.njk]: yes, its different "and any interest" is additional text for claim with intrest

File: src/main/common/components/ccj/paid-amount-summary.njk (hunk 1 above)
Comment (sudheer-ajax) [reply src/main/common/components/ccj/paid-amount-summary.njk]: must be a result of wrong keyboard shortcut :-)

**Pull request merged**